src/directives/markers.js

```diff
@@ -269,6 +269,9 @@ angular.module('ui-leaflet').directive('markers',
                             return;
                         _create(newMarkers, oldMarkers);
                     });
+                    scope.$on('$destroy', function () {
+                        _destroy(leafletScope.markers, {}, leafletMarkers, map, layers);
```

Comment:
I'm wondering if we should call deleteMarker on each leaflet marker when the directive is destroyed. Thus, we are sure that everything is reallyclean up